.github/workflows/linux_wheel.yml

```diff
@@ -0,0 +1,69 @@
+name: Build Linux Wheels
+
+on:
+  pull_request:
+
+jobs:
+  generate-build-matrix:
+    runs-on: ubuntu-latest
+    outputs:
+      matrix: ${{ steps.set-matrix.outputs.matrix }}
+    container:
+      image: python:3.9
+    steps:
+      - name: Clone pytorch/pytorch
+        uses: actions/checkout@v2
```

Comment:
For workflows that run on PRs, should we always configure them to checkout just the PR head rather than merging into `master` (the default behavior of `actions/checkout`), as discussed in #49578 and #49590?